src/pages/extending-chat-widget/custom-identity-provider/index.mdx

```diff
@@ -61,6 +61,8 @@ const customerSDK = CustomerSDK.init({
 })
 ```
 
+If you would like to learn more about setting up a Custom Identity Provider, please see our [tutorial](https://developers.livechat.com/updates/livechat-custom-identity-provider-setup-tutorial).
```

Comment:
I would create a separate # for it as it gets a little lost inside the document, perhaps something like that? I was also thinking to move it to the "Examples" section as it's more of an example of implementation, wdyt? I'd place it before the "Repository" section if we would move it there